src/cc/bcc_proc.c

```diff
@@ -17,20 +17,24 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <sys/mman.h>
+#include <ctype.h>
 #include <fcntl.h>
-#include <unistd.h>
-#include <stdlib.h>
+#include <limits.h>
+#include <math.h>
 #include <stdbool.h>
-#include <string.h>
 #include <stdint.h>
-#include <ctype.h>
 #include <stdio.h>
-#include <math.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
 
 #include "bcc_perf_map.h"
 #include "bcc_proc.h"
 #include "bcc_elf.h"
 
+// https://www.kernel.org/doc/Documentation/x86/x86_64/mm.txt
+const unsigned long long kernelAddrSpace = 0x00ffffffffffffff;
```

Comment:
How about other architectures?

Reply:
Good point. I couldn't find any similar file in other architecture's folder, so I will just gate this for x64